netbox_dns/choices/record.py

```diff
@@ -30,6 +32,20 @@ class RecordTypeChoices(ChoiceSet):
     ]
 
 
+@initialize_choice_names
+class RecordSelectableTypeChoices(ChoiceSet):
+    CHOICES = [
+        (rdtype.name, rdtype.name)
+        for rdtype in sorted(rdatatype.RdataType, key=lambda a: a.name)
+        if not rdatatype.is_metatype(rdtype)
+        and rdtype.name
+        not in get_plugin_config("netbox_dns", "filter_record_types", [])
```

Comment:
I'm not sure if we should allow or deny by default. Allowing has the advantage of covering future enhancements without changing the configuration. However, denying prevents new record types from being introduced for users without admin consent.

Reply:
IMHO, we definitely should deny by default, as the list of allowed types is inherited from `dnspython` and may change over time. Having an allow list in the configuration may lead to hard-to-trace problems if a newer `dnspython` version drops a record type that is on the allow list. A deny list is more forgiving in this case.